CHANGES.rst

```diff
@@ -6,7 +6,15 @@
 2.1a2 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Implemented the storage ``afterCompletion`` method, which allows
+  RelStorage storages to be notified of transaction endings for
+  transactions that don't call the two-phase commit API.  This allows
+  resources to be used more efficiently because it prevents RDBMS
+  transactions from being held open.
+
+  Fixes: https://github.com/zodb/relstorage/issues/147
```

Comment:
Wait, I spelled this the same way you did below (in CHANGES.rst and both are broken.

Why do you think this is a good idea? :-)